packages/camera/camera_android_camerax/android/src/test/java/io/flutter/plugins/camerax/CameraTest.java

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camerax;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import androidx.camera.core.Camera;
+import io.flutter.plugin.common.BinaryMessenger;
+import java.util.Objects;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.robolectric.RobolectricTestRunner;
+
+@RunWith(RobolectricTestRunner.class)
```

Comment:
Does this test need to be run with `RobolectricTestRunner`? I noticed the other tests have this as well. So if no, then they can remove it as well.